nipype/interfaces/base/boutiques.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+import os
+import json
+from collections import defaultdict
+
+from ... import logging
+from ..base import (
+    traits, File, Str, isdefined, Undefined,
+    DynamicTraitedSpec, CommandLine)
+
+iflogger = logging.getLogger('nipype.interface')
+
+
+class BoutiqueInterface(CommandLine):
+    """Convert Boutique specification to Nipype interface
+
+    """
+
+    input_spec = DynamicTraitedSpec
+    output_spec = DynamicTraitedSpec
+
+    # Subclasses may override the trait_map to provide better traits
+    # for various types
+    trait_map = {
+        'File': File,
+        'String': Str,
+        'Number': traits.Float,
```

Comment:
@gkiar - is it really only `Number`? shouldn't boutiques support at least the XSD literal datatypes?

Reply:
There is a separate "integer" constraint, as well.